### PR TITLE
Category touch up

### PR DESCRIPTION
**Pull request by ming-00**
Explore and category page should be more or less done~

Left 2 do: 
 - progress rings
 - think of styling the challenge icons

### CODE REVIEW

File: frontend/src/components/common/Searchbar.tsx [hunk 1]
```diff
@@ -3,6 +3,11 @@ import InputBase from '@mui/material/InputBase';
 import { styled, alpha } from '@mui/material/styles';
 import { ReactComponent as SearchIcon } from '../../assets/icons/search-icon.svg';
 
+interface SearchbarProps {
+  placeholder: string;
+  func: (e: any) => void;
```
Comment (sebastiantoh) [inline frontend/src/components/common/Searchbar.tsx:8]: maybe rename this to `onChange` or something? `e` can probably be typed as a string as well i think

File: frontend/src/pages/feed/FeedPage.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+const HomePage: React.FC = () => {
+  return (
+    <>
+      <Typography variant="h5">Upcoming :-)</Typography>
+      <img src="https://i.redd.it/ox49yfg9vn461.jpg" style={{ width: '80%' }} />
+    </>
+  );
+};
+
+export default HomePage;
```
Comment (sebastiantoh) [inline frontend/src/pages/feed/FeedPage.tsx:13]: should probably rename this to FeedPage?

File: frontend/src/pages/explore/ExplorePage.tsx [hunk 3]
```diff
@@ -1,39 +1,89 @@
...
-              />
-            </Link>
-          </li>
-        ))}
-      </ul>
+      <Searchbar
+        placeholder="Search by challenge name..."
+        func={eventhandler}
+      />
+      {value.length > 0 && (
```
Comment (sebastiantoh) [inline frontend/src/pages/explore/ExplorePage.tsx:44]: hmm, is it weird that the challenges appear when previously (before the search) it was the categories?

File: frontend/src/pages/explore/ExplorePage.tsx [hunk 4]
```diff
@@ -1,39 +1,89 @@
...
+import { loadAllChallenges } from 'store/challenges/operations';
+import { getChallengeList } from 'store/challenges/selectors';
+
 import './ExplorePage.scss';
 
 const ExplorePage: React.FC = () => {
   // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
   const categories = useSelector((state: RootState) => getCategoryList(state))!;
+  const dispatch = useDispatch();
+  const [value, setValue] = useState('');
```
Comment (sebastiantoh) [inline frontend/src/pages/explore/ExplorePage.tsx:22]: can probably rename this to `searchQuery` or something

**Review by sebastiantoh — COMMENTED** [review]
looks good! just some nitpicky renaming comments

File: frontend/src/pages/explore/ExplorePage.tsx (hunk 3 above)
Comment (ming-00) [reply frontend/src/pages/explore/ExplorePage.tsx:44]: I think Spotify explore page also does sth similar, w/o search it's genres and when you search it returns songs/ playlists/ artists

Comment (sebastiantoh) [reply frontend/src/pages/explore/ExplorePage.tsx]: I see! In that case I think the current implementation looks good!

**Comment by sebastiantoh** [comment]
lgtm!

**Pull request merged**